chia/wallet/cat_wallet/cat_utils.py

```diff
@@ -36,15 +36,16 @@ def match_cat_puzzle(mod: Program, curried_args: Program) -> Optional[Iterator[P
     if it is, return the curried arguments
     """
     if mod == CAT_MOD:
-        return curried_args.as_iter()
+        ret: Iterator[Program] = curried_args.as_iter()
```

Comment:
Note that `Program` is `Any` due to inheriting from `SExp` which is in the unhinted `clvm` and thus treated as `Any`.  https://github.com/Chia-Network/clvm/pull/102  No need to change here, just be aware that the hinting doesn't do much.